### PR TITLE
Update Thanos maintainers and add new Prometheus maintainers too

### PR DESCRIPTION
**Pull request by metalmatze**
Please take a look if your information is correct @Nexucis, @mxinden, @kakkoyun, @squat, @onprem, @yeya24.

Updates taken from the projects maintainer lists.

### CODE REVIEW

File: project-maintainers.csv [hunk 1]
```diff
@@ -396,9 +399,12 @@ Sandbox,OpenEBS,Amit Kumar Das,MayaData,AmitKumarDas,https://github.com/openebs/
 ,,Vishnu Itta,MayaData,vishnuitta,
 Incubating,Thanos,Bartłomiej Płotka,Red Hat,bwplotka,https://github.com/thanos-io/thanos/blob/master/MAINTAINERS.md
 ,,Frederic Branczyk ,Polar Signals,brancz,
-,,Giedrius Statkevičius,,GiedriusS,
-,,Povilas Versockas,,povilasv,
-,,Dominic Green,Improbable,domgreen,
+,,Giedrius Statkevičius,Vinted,GiedriusS,
+,,Kemal Akkoyun,Polar Signals,kakkoyun,
+,,Lucas Servén Marín,,squat,
```
Comment (GiedriusS) [inline project-maintainers.csv:404]: Maybe we could add `Red Hat` here? I think Prem & Lucas are working there unless something has changed very recently :smile:

Comment (metalmatze) [reply project-maintainers.csv:404]: Something recently changed for @squat. I don't know exactly what the new company is just yet, I pinged him about it.

Comment (squat) [reply project-maintainers.csv:404]: I left red hat a few months ago actually! I'm currently unaffiliated :)

Comment (squat) [reply project-maintainers.csv:404]: thanks for making this @metalmatze :)

**Review by bwplotka — APPROVED** [review]
LGTM, just @onprem is missing his affiliation! (:

File: project-maintainers.csv [hunk 2]
```diff
@@ -396,9 +399,12 @@ Sandbox,OpenEBS,Amit Kumar Das,MayaData,AmitKumarDas,https://github.com/openebs/
...
+,,Prem Saraswat,,onprem,
```
[the remark this answers is not in the document]
Comment (metalmatze) [reply project-maintainers.csv:405]: Oh, that indeed seems to be missed. Fixing.

**Review by Nexucis — APPROVED** [review]
cool ! thank you for doing that @metalmatze !

**Comment by amye** [comment]
We good to merge here?

**Review by squat — APPROVED** [review]

**Review by yeya24 — APPROVED** [review]

**Comment by metalmatze** [comment]
Yes, sound like we're ready! Thanks @amye!

**Review by mxinden — APPROVED** [review]
My data is correct. Thanks @metalmatze.

**Pull request merged**